examples/hitl/pick_throw_vr/config/pick_throw_vr.yaml

```diff
@@ -24,5 +24,9 @@ habitat_hitl:
   gui_controlled_agent:
     agent_index: 1
     ang_speed: 15
+  networking:
+    active_features:
+      # This is a first-person application. We don't need to transmit skinned mesh poses.
```

Comment:
Please add comments pointing people to `hide_humanoid_in_gui` and `camera.first_person_mode`.

I'm seeing that we're creating combinations of Hydra config fields that must be set jointly, otherwise they are somewhat invalid. I guess that is fine but we should at least add comments helping people connect the related config fields.